test-galasactl-ecosystem.sh

```diff
@@ -791,7 +791,546 @@ function launch_test_from_unknown_portfolio {
+        error "Failed to get properties whilst infix is set"
+        exit 1
+    fi
+    success "Properties get with infix set seems to be successful."
+}
+
+#--------------------------------------------------------------------------
+function properties_get_with_prefix_infix_and_suffix {
+    h2 "Performing properties get with prefix, infix, and suffix set, expecting list of properties..."
+    
+    cmd="${BASEDIR}/bin/${binary} properties get --namespace ecosystemtest \
+    --prefix get \
+    --suffix property \
+    --infix test \
+    --bootstrap $bootstrap \
+    --log -"
+
+    info "Command is: $cmd"
+
+    output_file="properties-get-output.txt"
+    $cmd | tee $output_file
+    rc=$?
+    if [[ "${rc}" != "0" ]]; then 
+        error "Failed to get properties with prefix, infix, and suffix set: command failed."
+        exit 1
+    fi
+
+    # Check that the previous properties set created a property
+    cat $output_file | grep "get.test.property this-shouldnt-be-deleted" -q -E
+
+    rc=$?
+    # We expect a return code of 0 because this is a properly formed properties get command.
+    if [[ "${rc}" != "0" ]]; then 
+        error "Failed to get properties whilst prefix, infix, and suffix is set"
+        exit 1
+    fi
+    success "Properties get with prefix, infix, and suffix set seems to be successful."
+}
+
+#--------------------------------------------------------------------------
+# function properties_get_with_name_prefix_infix_and_suffix {
+#     h2 "Performing properties get with name, prefix, infix, and suffix set, expecting an error..."
+    
+#     cmd="${BASEDIR}/bin/${binary} properties get --namespace ecosystemtest \
+#     --name get.test.property \
+#     --prefix get \
+#     --suffix property \
+#     --infix test \
+#     --bootstrap $bootstrap \
+#     --log -"
+
+#     info "Command is: $cmd"
+
+#     $cmd
+
+#     rc=$?
+#     if [[ "${rc}" != "1" ]]; then 
+#         error "Failed to recognise getting properties with name, prefix, infix, and suffix set should error."
+#         exit 1
+#     fi
+
+#     success "Properties get with name, prefix, infix, and suffix set seems to properly error."
+# }
+# TO BE ADDED IN ONCE PROPERTIES GET CHANGED TO NOT FAIL ON THIS TEST: ADD LOGIC IN TO MAKE FLAGS MUTUALLY EXCLUSIVE + UNIT TEST
+
+function properties_tests {
+    get_random_property_name_number
+    properties_create
+    properties_update
+    properties_delete
+    properties_delete_invalid_property
+    properties_delete_without_name
+    properties_set_with_name_without_value
+    properties_set_without_name_with_value
+    properties_set_without_name_and_value
+    properties_get_setup
+    properties_get_with_namespace
+    properties_get_with_name
+    properties_get_with_prefix
+    properties_get_with_suffix
+    properties_get_with_infix
+    properties_get_with_prefix_infix_and_suffix
+    #properties_get_with_name_prefix_infix_and_suffix
+}
+
+ calculate_galasactl_executable
```

Comment:
I'd be tempted to move this out into a second script.
Then use this:
```
source ${BASEDIR}/that-subscript.sh
```

Then you can invoke properties_test from the bigger file.